PWGCF/Tasks/CMakeLists.txt

```diff
@@ -29,3 +29,7 @@ o2physics_add_dpl_workflow(correlations
                     PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore O2Physics::PWGCFCore
                     COMPONENT_NAME Analysis)
 
+o2physics_add_dpl_workflow(flow-generic-framework
+                    SOURCES flowgenericframework.cxx
```

Comment:
There is nothing wrong with Capitals in the filename, just then not in the executable name. So each capital becomes a dash. I.e.:
flowGenericFramework -> flow-generic-framework
You can google camel case vs snake case for your amusement :)